### DISCUSSION TITLE
mlflow.evaluate: Subset evaluation metrics?

### DISCUSSION THREAD

**Discussion by khojarohan4** [mlflow/mlflow#6367] [Q&A] [answered]
Is there a way, we can subset the list of metrics present for "default" evaluator?

 result = mlflow.evaluate(model_info.model_uri,
        eval_data,
        targets="target",
        model_type="classifier",
        dataset_name="abc",
        evaluators=["default"],
        evaluator_config={"default":{"log_model_explainability":False}},
        custom_metrics=[example_custom_metric_fn],
    )

For default evaluators, we have many metrics captured but is there a way, we could limit the number of metrics? I could define my own I understand, just checking if there is straight forward way.

**Comment by WeichenXu123** [comment, marked as answer]
For now we can't, I think this is a good proposal. Could you file a feature request ticket for this ? :)
CC @dbczumar

**Reply by khojarohan4** [reply]
@WeichenXu123 Do I create issue from this discussion and label it as feature request/Improvement rather than it been seen as an issue? This is my first so spare me the blushes :P

**Reply by WeichenXu123** [reply]
You should create it by https://github.com/mlflow/mlflow/issues/new?assignees=&labels=enhancement&template=feature_request_template.yaml&title=%5BFR%5D
and fill the template.

**Reply by khojarohan4** [reply]
Done.
